### DISCUSSION TITLE
How to Install QT Version in Project64 3.0?

### DISCUSSION THREAD

**Discussion by IsaacGibsound** [gonetz/GLideN64#2885] [Q&A]
Hey, GLideN64 community!

I have a question whose answer may be a bit obvious, but I would still like help from y'all because as of yet, I have been unable to figure it out. I want to download and install a QT version of GLideN64 in Project64 3.0.1-5664-2df3434. My reason is that I saw a comment here on GitHub under [Issues](https://github.com/gonetz/GLideN64/issues) saying how to fix a certain graphical glitch in Mario Kart 64, but it requires changing a setting that is only changeable in some certain version(s) of GLideN64. ([Here's a link to the comment, if you are interested.](https://github.com/gonetz/GLideN64/issues/2506#issuecomment-850476206)) The particular GLideN64 version that this commenter was using is QT version [15668b9](https://github.com/gonetz/GLideN64/commit/15668b9064bbc0aca524d9a74ece98aaa9260908). So, the title basically says it all: I would like to know how to download and install a QT version of GLideN64 (such as the aforementioned version) in Project64 3.0.1-5664-2df3434.

Thank you in advance,
Isaac "Gibsound"

**Comment by gonetz** [comment]
To download:
The main page of the project, https://github.com/gonetz/GLideN64
contains "Continuous Integration" section with download links.
Go to Releases, https://github.com/gonetz/GLideN64/releases/tag/github-actions
Find proper build here and download it. It should be *-Windows-Project64-Qt-x64.zip for 64bit Project64 or *-Windows-Project64-Qt-x86.zip for 32bit Project64.
To install:
The .zip file should be unpacked to the Plugin\GFX\GLideN64 sub-folder of your Project64 folder, with replacement of original files. Select GLideN64 as the graphics plugin in Project64 settings.

**Reply by petran79** [reply]
> To download: The main page of the project, https://github.com/gonetz/GLideN64 contains "Continuous Integration" section with download links. Go to Releases, https://github.com/gonetz/GLideN64/releases/tag/github-actions Find proper build here and download it. It should be *-Windows-Project64-Qt-x64.zip for 64bit Project64 or *-Windows-Project64-Qt-x86.zip for 32bit Project64. To install: The .zip file should be unpacked to the Plugin\GFX\GLideN64 sub-folder of your Project64 folder, with replacement of original files. Select GLideN64 as the graphics plugin in Project64 settings.

I follow the steps in PJ64 3.0 but no matter the version (QT or WTL), the emulator does not recognize GlideN64 and it is not listed in the plugins folder.
